mailpoet/lib/Services/Bridge.php

```diff
@@ -125,7 +125,7 @@ public function getAuthorizedEmailAddresses($type = 'authorized'): array {
     if ($data && $type === 'all') {
       return $data;
     }
-    return $data ? $data[$type] : [];
+    return $data && isset($data[$type]) ? $data[$type] : [];
```

Comment:
I don't think that part of the condition is redundant.

Unlike the previous implementation, this includes `$data[$type]` where `$type` is now a variable. 

It's a good idea to confirm the value exists before accessing it otherwise we would get  an Undefined index error

Reply:
@triple0t I still think the condition would work without the first part (`$data`).
Take look at [#](https://onlinephp.io/?s=s7EvyCjg5eLlUimpLEhVsFVQT0lNy8xLTVG35uXS11dQSUksSQQKR8PFFWztgLxEdR0F9ST12FigsrLEoviU0twCjczi4tQSDbCWaLB5sZoK9grIfAUrhehYTWsA&v=7.4.30).

Reply:
Okay 👍